commands/service_list.go

```diff
@@ -44,18 +45,23 @@ func (c *serviceListCmd) runE(cmd *cobra.Command, args []string) error {
 	}
 
 	w := tabwriter.NewWriter(os.Stdout, 0, 0, 4, ' ', 0)
-	fmt.Fprintf(w, "SERVICE\tNAME\n")
 
-	// TODO: implement status to ServiceList API.
-	//fmt.Fprintf(w, "STATUS\tSERVICE\tNAME\n")
+	fmt.Fprintf(w, "STATUS\tSERVICE\tNAME\t\n")
 	for _, s := range services {
-		// status, err := s.Status()
-		// if err != nil {
-		// 	return err
-		// }
-
-		fmt.Fprintf(w, "%s\t%s\n", s.ID, s.Name)
-		//fmt.Fprintf(w, "%s\t%s\t%s\n", status.String(), s.ID, s.Name)
+		var status string
```

Comment:
The service has [String](https://github.com/mesg-foundation/core/blob/3ae39191f1b216586938d30bb764084295bb64f1/protobuf/coreapi/api.pb.go#L105) method, why you recreated it here (and also there is "semi" implementation in the _test.go file)

Reply:
I didn't know that, thanks for the heads up!